examples/foam/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "private": true,
+  "dependencies": {
+    "todomvc-common": "^1.0.0",
+    "todomvc-app-css": "^1.0.1"
+  },
+  "devDependencies": {
+    "foam-framework": "0.4.2"
+  }
```

Comment:
Yes, it's true. You need FOAM to run the `build.sh` script which constructs `build/foam.js`. That file contains the app code and the subset of the framework needed to run it.